### PR TITLE
Adding a new VM to fakeovirt for insecure test

### PR DESCRIPTION
**Pull request by bkhizgiy**

### CODE REVIEW

File: stubs/vms/4db71538-9255-4ba3-a291-1240682dd8c6/content.json [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+        "rel" : "undosnapshot"
+      }, {
+        "href" : "/ovirt-engine/api/vms/4db71538-9255-4ba3-a291-1240682dd8c6/cancelmigration",
+        "rel" : "cancelmigration"
+      }, {
+        "href" : "/ovirt-engine/api/vms/4db71538-9255-4ba3-a291-1240682dd8c6/export",
+        "rel" : "export"
+      } ]
+    },
+    "name" : "vm-for-migration",
```
Comment (ahadas) [inline stubs/vms/4db71538-9255-4ba3-a291-1240682dd8c6/content.json]: we don't support two vms with the same name within the same cluster in ovirt so let's give it a different name (I guess we'll fail to import both to the same namespace in k8s)

Comment (bkhizgiy) [reply stubs/vms/4db71538-9255-4ba3-a291-1240682dd8c6/content.json]: ack, changed to `vm-for-insecure-migration`

**Comment by liranr23** [comment]
what i'm afraid from is imagetransfer when triggering multiple tests to the same endpoints, most of them are ok but the `/ovirt-engine/api/imagetransfers/abc123` does switch. i think one test will pass and the other won't if we use the same endpoints.
did you check it? if it won't work, i think either another endpoint should be added or deleting the fakeovirt pod to refresh it in the test env.

File: stubs/imagetransfers/stub.json [hunk 2]
```diff
@@ -32,28 +32,76 @@
...
       {
-        "responseBody": "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?><image_transfer href=\"/ovirt-engine/api/imagetransfers/abc123\" id=\"abc123\"><direction>download</direction><format>raw</format><phase>finished_success</phase><transfer_url>http://ovirt-imageio.konveyor-forklift:12345/images/cirros</transfer_url><host href=\"/ovirt-engine/api/hosts/2900d577-63a8-4ec9-acff-8493278346b1\" id=\"2900d577-63a8-4ec9-acff-8493278346b1\"/></image_transfer>",
+        "responseBody": "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?><image_transfer href=\"/ovirt-engine/api/imagetransfers/def456\" id=\"def456\"><direction>download</direction><format>raw</format><phase>finished_success</phase><transfer_url>http://ovirt-imageio.konveyor-forklift:12345/images/cirros</transfer_url><host href=\"/ovirt-engine/api/hosts/2900d577-63a8-4ec9-acff-8493278346b1\" id=\"2900d577-63a8-4ec9-acff-8493278346b1\"/></image_transfer>",
         "responseCode": 200,
+    "times": 10000
+      }
+    ]
+  },
+  {
+    "path": "/ovirt-engine/api/imagetransfers",
```
Comment (bkhizgiy) [inline stubs/imagetransfers/stub.json]: @liranr23 I've added a new stub for different image transfer endpoints, but I'm not so sure about the post operation, we will need somehow distinguish between the secure and insecure flows to know which transfer to create. same here: https://github.com/kubev2v/fakeovirt/blob/5104cfaa997a345e0e69a9ad702e6d73d5e360dc/internal/app/handlers/static/static-handler.go#L120

**Comment by bkhizgiy** [comment]
Was tested locally and the test working now with the new stubs.

**Review by ahadas — APPROVED** [review]

**Pull request merged**